eos_data_distribution/ndn/base.py

```diff
@@ -330,7 +330,7 @@ def removePendingInterest(self, name):
     def onTimeout(self, interest, forever=False):
         name = interest.getName()
         self._callbackCount += 1
-        self.emit('interest-timeout', interest)
+        self.emit('interest-timeout', interest, forever)
```

Comment:
Whoops. Fixed the commit message, and will add a follow-up patch to rename `forever` to `try_again` everywhere.